rbs/prism/AssertionsRewriterPrism.cc

```diff
@@ -224,23 +217,9 @@ pm_node_t *deepCopyGenericTypeNode(parser::Prism::Parser &parser, pm_node_t *nod
 
         case PM_ARGUMENTS_NODE: {
             auto *original = down_cast<pm_arguments_node_t>(node);
-            auto *copy = prism.allocateNode<pm_arguments_node_t>();
-
-            size_t argCount = original->arguments.size;
-            pm_node_t **copiedNodes = nullptr;
-
-            if (argCount > 0) {
-                copiedNodes = (pm_node_t **)prism.calloc(argCount, sizeof(pm_node_t *));
-                for (size_t i = 0; i < argCount; i++) {
-                    copiedNodes[i] = deepCopyGenericTypeNode(parser, original->arguments.nodes[i]);
-                }
-            }
+            auto originalArgs = absl::MakeSpan(original->arguments.nodes, original->arguments.size);
```

Comment:
This wasn't previously possible, if `createArgumentsNode()`​ required a `std::vector`​ as input. We'd have to waste time&space to copy out of the `pm_node_list_t`​, only for `createArgumentsNode()` to copy data out of it.  
  
But now that it takes a span, we can just give a few into the original `pm_node_list_t`​, and have `createArgumentsNode()`​ copy directly out of that :+1: